src/components/pages/FamilylistTable/FamilyListTable.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Table } from 'antd';
+
+const FamilyListTable = ({ family }) => {
+  const data = family.map(oneFamily => {
+    return {
+      key: oneFamily.id,
+      family_id: oneFamily.id,
+      name: oneFamily.name,
+      email: oneFamily.email,
+      case_manager: oneFamily.username,
+    };
+  });
+
+  //console.log('familyList', data);
```

Comment:
Great job, Polly! I would just delete this commented-out console log since it's not necessary.

Reply:
Thank you for your suggestion, Melenie. I just deleted it.